.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,66 @@
+name: Feature Request
+description: Request a new feature to help your project using the SAP Cloud SDK (JavaScript / TypeScript).
+title: '<title>'
+labels: 
+  - feature request
+body:
+  - type: markdown
+    attributes:
+      value: |
+        > [!note]
+        > Please note that this repository is for JavaScript / TypeScript related issues only. 
+        > If you have a question about the SAP Cloud SDK (Java), open a GitHub issue here: https://github.com/SAP/cloud-sdk-java/issues/new/choose
```

Comment:
[q] Should we just hide this like in the original version? I feel like we've not had any issues where people were confused. This is just taking more space. No strong opinions though.

Reply:
It was not really hidden previously. I would just leave it there.

![image](https://github.com/user-attachments/assets/4a81573f-d758-4d7c-8134-c8f922edaba3)